### DISCUSSION TITLE
blinkyparts OpenDTU - NRF24 not connected

### DISCUSSION THREAD

**Discussion by fouille** [tbnobody/OpenDTU#1202] [Q&A] [answered]
Hello, 

I just bought a kit for openDTU, which I assembled with the greatest care despite my beginnings in soldering :-)

Everything seems to be working fine except the most important the NRF24 probe :-( in the interface it tells me that it is not connected...

**my actions:**
* read the different tutorials and QA
* modify for a more resent version the pin_mapping.json
* test with opendtu_generic.bin firmware
* update (today's version)
* change electrical adapter and test on Mac USB port.
* change cord

Nothing helps.
Do you have an idea to help me please?
PS (I'm a beginner) :-) 

![Capture d’écran 2023-08-02 à 23 25 45](https://github.com/tbnobody/OpenDTU/assets/24554441/c036f901-c475-4d83-8dae-1333fbcbc492)

![Capture d’écran 2023-08-02 à 23 25 20](https://github.com/tbnobody/OpenDTU/assets/24554441/ea91b2d9-3938-4135-877d-ff2e52d23ff4)

```json
[
    {
        "name": "NRF, LEDs, Display",
        "nrf24": {
            "miso": 19,
            "mosi": 23,
            "clk": 18,
            "irq": 16,
            "en": 4,
            "cs": 5
        },
        "display": {
            "type": 3,
            "data": 21,
            "clk": 22
        },
        "led": {
            "led0": 25,
            "led1": 26
        }
    },
    {
        "name": "CMT, LEDs, Display",
        "nrf24": {
            "miso": -1,
            "mosi": -1,
            "clk": -1,
            "irq": -1,
            "en": -1,
            "cs": -1
        },
        "cmt": {
            "clk": 18,
            "cs": 4,
            "fcs": 5,
            "sdio": 23,
            "gpio2": 19,
            "gpio3": 16
        },
        "display": {
            "type": 3,
            "data": 21,
            "clk": 22
        },
        "led": {
            "led0": 25,
            "led1": 26
        }
    },
    {
        "name": "NRF, Display",
        "nrf24": {
            "miso": 19,
            "mosi": 23,
            "clk": 18,
            "irq": 16,
            "en": 4,
            "cs": 5
        },
        "display": {
            "type": 3,
            "data": 21,
            "clk": 22
        }
    },
    {
        "name": "CMT, Display",
        "nrf24": {
            "miso": -1,
            "mosi": -1,
            "clk": -1,
            "irq": -1,
            "en": -1,
            "cs": -1
        },
        "cmt": {
            "clk": 18,
            "cs": 4,
            "fcs": 5,
            "sdio": 23,
            "gpio2": 19,
            "gpio3": 16
        },
        "display": {
            "type": 3,
            "data": 21,
            "clk": 22
        }
    },
    {
        "name": "NRF, LEDs",
        "nrf24": {
            "miso": 19,
            "mosi": 23,
            "clk": 18,
            "irq": 16,
            "en": 4,
            "cs": 5
        },
        "led": {
            "led0": 25,
            "led1": 26
        }
    },
    {
        "name": "CMT, LEDs",
        "nrf24": {
            "miso": -1,
            "mosi": -1,
            "clk": -1,
            "irq": -1,
            "en": -1,
            "cs": -1
        },
        "cmt": {
            "clk": 18,
            "cs": 4,
            "fcs": 5,
            "sdio": 23,
            "gpio2": 19,
            "gpio3": 16
        },
        "led": {
            "led0": 25,
            "led1": 26
        }
    },
    {
        "name": "NRF",
        "nrf24": {
            "miso": 19,
            "mosi": 23,
            "clk": 18,
            "irq": 16,
            "en": 4,
            "cs": 5
        }
    },
    {
        "name": "CMT",
        "nrf24": {
            "miso": -1,
            "mosi": -1,
            "clk": -1,
            "irq": -1,
            "en": -1,
            "cs": -1
        },
        "cmt": {
            "clk": 18,
            "cs": 4,
            "fcs": 5,
            "sdio": 23,
            "gpio2": 19,
            "gpio3": 16
        }
    }
]
```

**Comment by stefan123t** [comment]
duplicate / similar to this https://github.com/tbnobody/OpenDTU/discussions/1132

**Comment by tbnobody** [comment]
Could you show us a picture of your soldered PCB?

**Comment by fouille** [comment]
![IMG_1989](https://github.com/tbnobody/OpenDTU/assets/24554441/3290fb7d-f1e2-4ef8-84d3-11988622d1c1)
![IMG_1992](https://github.com/tbnobody/OpenDTU/assets/24554441/6bcfce16-025f-4064-a6af-59cc527f951a)
![IMG_1991](https://github.com/tbnobody/OpenDTU/assets/24554441/e6ae2360-6d67-4c6b-b9d0-dfc178ced98c)
![IMG_1990](https://github.com/tbnobody/OpenDTU/assets/24554441/8383097c-f71b-4595-b310-bd2c1cbdf22f)

**Reply by Sprinterfreak** [reply]
@fouille please re-solder some blobs. There's definitly points which don't make a good connection.
Quick tip for soldering: Make the iron touch the circuit-board and the wire to solder at the same time. Both the board and the wire must be sufficiantly hot for the solder to bond to both surfaces to form not only a mechanical but an electrical connection.
The pictures look like You only touched the legs of the components with Your Iron.

**Reply by fouille** [reply]
I redid all my points, they are in good contact with the PCB.
can you tell me if you see an anomaly on the photos?
(it's not too pretty I know :-) it's a first for me :-) )

![IMG_1997](https://github.com/tbnobody/OpenDTU/assets/24554441/fba9a7d8-078c-42c8-bfb3-e0cc276c5a09)
![IMG_1996](https://github.com/tbnobody/OpenDTU/assets/24554441/784e6a92-3d80-4233-b3fb-d37f741eb4d4)

**Comment by fouille** [comment, marked as answer]
I found my error, I had a contact point between two welds here, now it is well detected. Thanks to the zoom on the photos I could see it :-)

![Capture d’écran 2023-08-03 à 01 06 53](https://github.com/tbnobody/OpenDTU/assets/24554441/c10d604d-e495-48fd-a635-58c730893549)

Thank's at all !
